src/app/onboarding/components/welcome-screen/welcome-screen.component.html

```diff
@@ -1,16 +1,32 @@
 <div class="welcome-screen">
-  <p *ngIf="pendingArchives.length === 0" class="welcome-message">Welcome to Permanent! Insert more welcome text here.</p>
-  <p *ngIf="pendingArchives.length > 0" class="welcome-message">Welcome to Permanent! Get started by accepting an invitation or creating a new archive.</p>
-  <div class="pending-archive-list" style="max-width: 800px; text-align: left; margin: 0 auto;" *ngIf="pendingArchives.length > 0">
-    <strong>Pending Archives:</strong>
-    <pr-archive-small *ngFor="let archive of pendingArchives"
-    [largeOnDesktop]="true" [archive]="archive"
-    [accessRole]="archive.accessRole"
-    [showAccept]="true"
-    (acceptClick)="acceptPendingArchive(archive)"
-    [showRemove]="false"></pr-archive-small>
-    <br />
-    <br />
+  <h2>Welcome to Permanent!</h2>
+  <div *ngIf="pendingArchives.length === 0" class="welcome-message no-invitations">
+    <p>You’ve just taken the first step on a journey to preserving what matters the most to you by creating archives that can represent yourself, your family, or your community.</p>
+
+    <p>Next, you’ll create your first archive. A Permanent archive is the collection of digital materials about an individual, family or group, or organizational entity.</p>
+    <div class="get-started">
+      <button class="btn btn-primary" (click)="goToScreen(OnboardingScreen.newArchive)">Get Started</button>
+    </div>
+  </div>
+  <div *ngIf="pendingArchives.length > 0" class="welcome-message has-invitations">
+    <p>You’ve been invited to collaborate on an archive as an archive member. A Permanent archive is the collection of digital materials about an individual, family or group, or organizational entity. Get started by accepting an invitation, or by creating a new archive of your own.</p>
+
+    <div class="pending-archive-list" style="margin: 0 auto;">
+      <strong>Pending invitations:</strong>
+      <pr-archive-small *ngFor="let archive of pendingArchives"
+      [largeOnDesktop]="true" [archive]="archive"
+      [accessRole]="archive.accessRole"
+      [showAccept]="true"
+      [showAcceptMobile]="true"
+      [showDefault]="false"
+      (acceptClick)="acceptPendingArchive(archive)"
+      [showRemove]="false"></pr-archive-small>
+      <br />
+      <br />
+    </div>
+
+    <div class="create-archive">
+      <a href="javascript:;" (click)="goToScreen(OnboardingScreen.newArchive)">Create a new archive</a>
+    </div>
```

Comment:
I don't see any corresponding CSS for `.create-archive`, so this is rendering pretty plainly.

Reply:
I checked on dev, and this is not just my broken build! :sweat_smile: 

![Screen Shot 2022-02-24 at 14 11 11](https://user-images.githubusercontent.com/1494855/155616143-ea13871c-a7d2-4398-94fc-2f4b7177dc27.png)

Reply:
That link is plainly styled on the Figma designs. I think because it's a more "minor" action compared to accepting the invitations. In the Figma, there's also an "Accept All" button to direclty contrast with it, which I believe we've agreed is not part of the MVP for this flow.

Reply:
Oh! So it is. Sorry, and nevermind, then!